### DISCUSSION TITLE
How to get project directory in Parcel 2 plugin?

### DISCUSSION THREAD

**Discussion by LandonSchropp** [parcel-bundler/parcel#5211] [Q&A] [answered]
I'm writing a Nunjucks transformer for Parcel 2, and I need to specify some configuration relative to the project directory for the consumer of the plugin. Is there a property or method built-in to parcel somewhere so I can easily determine the root directory of the project?

**Comment by devongovett** [comment, marked as answer]
`options.projectRoot` should give you what you want!

**Reply by LandonSchropp** [reply]
Thank you!

**Reply by Frantisek4187** [reply]
🤣🤣🤣
